core/src/main/java/org/apache/spark/memory/TaskMemoryManager.java

```diff
@@ -277,10 +311,20 @@ public long cleanUpAllAllocatedMemory() {
         freedBytes += memory.size();
         // We don't call free() here because that calls Set.remove, which would lead to a
         // ConcurrentModificationException here.
-        executorMemoryManager.free(memory);
+        memoryManager.tungstenMemoryAllocator().free(memory);
```

Comment:
"Everywhere" is only five places, all in TaskMemoryManager. To address your comment, I think I'd have to add two methods, `allocateTungstenMemory()` and `freeTungstenMemory()`, which only forwarded to that field. Given that, I'm not sure that I want to fix this; it's a net increase in code and I don't feel that it necessarily buys us any increase in clarity or correctness.